### PR TITLE
docs(falco-driver-loader): add some comments in `falco-driver-loader`

### PR DESCRIPTION
**Pull request by Andreagit97**
Signed-off-by: Andrea Terzolo <andrea.terzolo@polito.it>

**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

/area engine

**What this PR does / why we need it**:

This PR enhances the information printed by the `falco-driver-loader` during the loading phase. This is just a proposal, i would love to hear some ideas on that

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
docs(falco-driver-loader): add some comments in `falco-driver-loader`
```

### CODE REVIEW

**Comment by Andreagit97** [comment]
Some examples of possible new outputs:

**BPF**

```
* Running falco-driver-loader for: falco version=0.32.1-31+928d322, driver version=b4c198773bf05486e122f6d3f7f63be125242413, arch=x86_64, kernel release=5.15.0-33-generic, kernel version=34
* Running falco-driver-loader with: driver=bpf, compile=yes, download=yes
* Filename 'falco_ubuntu-generic_5.15.0-33-generic_34.o' is composed of:
 - driver name: falco
 - target identifier: ubuntu-generic
 - kernel release: 5.15.0-33-generic
 - kernel version: 34
```

**Kernel module**

```
* Running falco-driver-loader for: falco version=0.32.1-31+928d322, driver version=b4c198773bf05486e122f6d3f7f63be125242413, arch=x86_64, kernel release=5.15.0-33-generic, kernel version=34
* Running falco-driver-loader with: driver=module, compile=yes, download=yes
* Filename 'falco_ubuntu-generic_5.15.0-33-generic_34.ko' is composed of:
 - driver name: falco
 - target identifier: ubuntu-generic
 - kernel release: 5.15.0-33-generic
 - kernel version: 34
```

File: scripts/falco-driver-loader [hunk 1]
```diff
@@ -504,15 +513,17 @@ load_bpf_probe_download() {
 }
 
 load_bpf_probe() {
-	echo "* Mounting debugfs"
 
 	if [ ! -d /sys/kernel/debug/tracing ]; then
+		echo "* Mounting debugfs"
```
Comment (Andreagit97) [inline scripts/falco-driver-loader:518]: This should be printed only if we really mount the `debugfs`

**Review by leogr — APPROVED** [review]
I like it!

**Review by jasondellaluce — APPROVED** [review]
/approve

Thanks!

**Pull request merged**